src/stream/src/executor/monitor/streaming_stats.rs

```diff
@@ -376,6 +377,19 @@ impl StreamingMetrics {
         )
         .unwrap();
 
+        let join_matched_join_keys_opts = histogram_opts!(
+            "stream_join_matched_join_keys",
+            "The number of keys matched in the opposite side",
```

Comment:
Not make sense, but can not come up with a better idea 😇

Reply:
how